xgi/classes/hypergraph.py

```diff
@@ -351,33 +367,20 @@ def remove_nodes_from(self, nodes):
 
         Parameters
         ----------
-        nodes : iterable container
-            A container of nodes (list, dict, set, etc.).  If a node
-            in the container is not in the graph it is silently
-            ignored.
+        nodes : iterable
+            An iterable of nodes.
 
         See Also
         --------
         remove_node
+
         """
         for n in nodes:
-            try:
-                edge_neighbors = self._node[n]
-                # list handles self-loops (allows mutation)
-                del self._node[n]
-                del self._node_attr[n]
-                for edge in edge_neighbors:
-                    self._edge[edge].remove(n)  # remove all edges n-u in graph
-                    # delete empty edges
-                    if len(self._edge[edge]) == 0:
-                        del self._edge[edge]
-                        del self._edge_attr[edge]
-            except KeyError as e:
-                pass
+            self.remove_node(n)
```

Comment:
This used to fail silently, so maybe in the for loop have
```
for n in nodes:
    if n in self._node:
        self.remove_node(n)
```

Reply:
That makes sense. I like the idea of throwing a warning in an `else` statement so that the method still runs, but the user is aware that there might be missing nodes. I'm fine with whatever you decide, though.